### DISCUSSION TITLE
[Abwesenheitsverwaltung] Abwesenheit Lehrer

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#15] [User Stories]
### User Story:
Als Lehrer möchte ich meine Abwesenheit über die Website können, um die Schule automatisch zu informieren und Zeit zu sparen.

### Beschreibung:
Es sollte eine eigene Seite geben auf der Lehrer sich als abwesend melden können und einen Zeitraum festlegen in welchem sie nicht in der Lage sind eine Schulstunde zu übernehmen. Die Schule kann so schnell reagieren und für die Zeit des Ausfalls planen.

**Optional:**
- Oder statt einer eigenen Seite nur ein kleiner Bereich / Button auf einer anderen Seite

### Akzeptanz-Kriterien:

- [ ] Lehrer hat die Möglichkeit sich abzumelden über die Website
- [ ] Die Abmeldung umfasst auch einen Zeitraum für die Abwesenheit 
- [ ] Die Abmeldung wird einem Admin gemeldet

**Comment by laralngn** [comment]
Vielleicht ändern zu allgemeinen Änderungen wie Vertretung, Raumänderung oder im schlimmsten Fall halt Entfall.

**Reply by Kronprinz03** [reply]
Ja allerdings habe ich mir halt gedacht das aus der Sicht des Lehrers, er / sie sich krankmeldet und die Schule schaut dann was getan werden kann Vertretung oder Ausfall. Raumänderung wäre ja wenn dann ein Teil der Vertretung und das dürfte ich auch in einem anderen Issue angesprochen haben aus der Sicht des Admins, der ja sich dann darum kümmern muss https://github.com/SE-TINF22B2/G2-Schoolify/discussions/43.
